jobspec2/addrs/README.md

```diff
@@ -0,0 +1 @@
+This package is copied from Packer: https://github.com/hashicorp/packer/tree/2bf912bddf297c907deef286b1d63dcd07e2c6c2/hcl2template/addrs
```

Comment:
I guess this is small enough that doing this outweighs the pain of having a subpackage import like we've done (and had pain around) for Consul?